### DISCUSSION TITLE
Plugins dont work when using a modelspec with parameters and endpoints menu disabled

### DISCUSSION THREAD

**Discussion by yashsinha-aero** [danny-avila/LibreChat#4110] [Troubleshooting] [answered]
### What happened?

I am using an OpenAI reverse proxy, with a custom model spec, for a gpt-4o model. Whenever i have parameters disabled ie parameters : false, i get a 404 error. The moment i turn it on, the request goes through. Any idea what might be causing this?

### Steps to Reproduce

1. Use custom model spec with endpoint as gptPlugins
2. Use an OpenAI reverse proxy (OPENAI_REVERSE_PROXY)
3. Add a preset tool, like tools :[“dalle”]
4. Set parameters : false

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by yashsinha-aero** [comment, marked as answer]
The fix i figured out with the help of the community was to add agentParams to the modelSpec. I am not sure how that makes sense, but it worked.

**Reply by ogipogi** [reply]
Hi can you elaborate what you exactly configured?
I have the same issue...

**Reply by yashsinha-aero** [reply]
Check the agent params docs for the librechat.yaml and add it to your modelspec.
